### PR TITLE
Fix list of allocations in the admin section(#78)

### PR DESCRIPTION
**Pull request by Isaac-alencar**
### Ensure the list of allocations will display active users only

Fixing a bug in admin section `/admin/allocations` that display inactive users on allocation's list.

To fix this I had to insert  a new ```where()``` in the following scopes on `app/models/allocation.rb:`
``` 
 :ongoing and :finished
```
and update  

- `app/queries/allocations_and_unalocated_users_query.rb`  to be able to filter by active users only

### CODE REVIEW

**Comment by adbatista** [comment]
missing specs

**Comment by Isaac-alencar** [comment]
I'll work on it, thanks for the warning

File: spec/models/allocation_spec.rb [hunk 1]
```diff
@@ -114,6 +114,58 @@
...
+               user: john,
+               company: company)
+        create(:allocation,
+               start_at: 3.months.after,
+               end_at: 4.months.after,
+               user: alex,
+               company: company)
+      end
+
+      it "return only allocation from active users only" do
```
Comment (wenderjean) [inline spec/models/allocation_spec.rb:140]: It should be `returns`

File: spec/models/allocation_spec.rb [hunk 2]
```diff
@@ -114,6 +114,58 @@
...
+              user: john,
+              company: company)
+        create(:allocation,
+              start_at: 10.day.ago,
+              end_at: 1.day.ago,
+              user: alex,
+              company: company)
+      end
+
+      it "return only allocation from active users only" do
```
Comment (wenderjean) [inline spec/models/allocation_spec.rb:163]: The same about using third person conjugation

File: spec/queries/allocations_and_unalocated_users_query_spec.rb [hunk 3]
```diff
@@ -72,5 +72,31 @@
         expect(names).to eq ['Alex Doratov', 'John Doe', 'Brian May', 'Roan Britt']
       end
     end
+
+    context 'when the user are inactive' do
```
Comment (wenderjean) [inline spec/queries/allocations_and_unalocated_users_query_spec.rb:76]: I believe you meant `when the users are inactive`

File: spec/features/admin/allocations_spec.rb [hunk 4]
```diff
@@ -30,6 +30,28 @@
...
+            start_at: Date.new(2019, 6, 17),
+            user: john,
+            project: project,
+            company: company )
+    end
+
+    it "shuold not display inactive users" do
+      expect(page).not_to have_text('John Doe')
+    end
+
```
Comment (wenderjean) [inline spec/features/admin/allocations_spec.rb:52]: You have an extra line here.

File: spec/features/admin/allocations_spec.rb [hunk 5]
```diff
@@ -30,6 +30,28 @@
...
+      visit '/admin/allocations'
+
+      create(:allocation,
+            start_at: Date.new(2019, 6, 17),
+            user: john,
+            project: project,
+            company: company )
+    end
+
+    it "shuold not display inactive users" do
```
Comment (wenderjean) [inline spec/features/admin/allocations_spec.rb:49]: You have a typo here, it should be `should`

File: spec/features/admin/allocations_spec.rb [hunk 6]
```diff
@@ -30,6 +30,28 @@
     end
   end
 
+  describe 'List of Allocations' do
+    let!(:john) {
```
Comment (wenderjean) [inline spec/features/admin/allocations_spec.rb:34]: It is not that common to use brackets for multi-line blocks, it is mostly used `do..end`

**Pull request closed without merging**